tests/unit_tests/all/plugins/test_export_runpath.py

```diff
@@ -3,19 +3,17 @@
 
 import pytest
 
-from ert.enkf_main import EnKFMain
 from ert.runpaths import Runpaths
 from ert.shared.hook_implementations.workflows.export_runpath import ExportRunpathJob
 from ert.shared.plugins import ErtPluginManager
-from ert.storage import open_storage
 
 
 @pytest.fixture
 def snake_oil_export_runpath_job(setup_case):
     ert_config = setup_case("snake_oil", "snake_oil.ert")
-    ert = EnKFMain(ert_config)
-    with open_storage(ert_config.ens_path, mode="w") as storage:
-        yield ExportRunpathJob(ert, storage)
+    plugin = ExportRunpathJob()
+    plugin.ert_config = ert_config
```

Comment:
Not a superfan of this direct assignment here, why can we not set it when we run it?